openedx/core/djangolib/tests/utils.py

```diff
@@ -0,0 +1,61 @@
+from django.core.cache import caches
+from django.test import TestCase, override_settings
+
+
+class CacheSetupMixin(TestCase):
```

Comment:
I think that there is content-specific caching that makes more sense to clear when the database clears. But I'm fine with leaving it as-is until we see that there's a problem.